packages/web-app-files/src/helpers/share/type.ts

```diff
@@ -35,12 +35,13 @@ export abstract class ShareTypes {
   static readonly link = new ShareType('link', 3, $gettext('Link'))
   static readonly guest = new ShareType('guest', 4, $gettext('Guest'))
   static readonly remote = new ShareType('remote', 6, $gettext('Federated'))
+  static readonly space = new ShareType('space', 7, $gettext('User'))
```

Comment:
I'm not sure 🤔  Technically, we're still talking about users. This change would result in "Member" being displayed under the username (instead of "User"):
![image](https://user-images.githubusercontent.com/50302941/156804788-5ca438f5-b3e5-41a1-9436-22b61a7617d3.png)

@tbsbdr What do you think?